.eslintrc

```diff
@@ -8,7 +8,7 @@
   "globals" : {
     "SERVER_ENABLED": true
   },
-  "parser": "babel-eslint",          // https://github.com/babel/babel-eslint
+  "parser": '@babel/eslint-parser',
```

Comment:
Super minor, but can we change this to be double quotes to be consistent with the rest of the file?

Reply:
Good catch. Dunno how this happened